src/app/app-modules/associate-anm-mo/video-consultation/video-consultation.component.ts

```
@@ -0,0 +1,92 @@
+} from '@angular/material/legacy-dialog';
+
+
+@Component({
+  selector: 'app-video-consultation',
+  templateUrl: './video-consultation.component.html',
+  styleUrls: ['./video-consultation.component.css']
+})
+
+export class VideoConsultationComponent {
+  // consent: boolean | null = null;
+  linkSent: boolean = false;
```

ESLint

[error] Type boolean trivially inferred from a boolean literal, remove type annotation.

(@typescript-eslint/no-inferrable-types)